src/game/gamecube_controller.c

```diff
@@ -193,6 +193,10 @@ s32 osContStartReadDataEx(OSMesgQueue* mq) {
     return ret;
 }
 
+s8 gGamecubeControllerCentersInitialized = FALSE;
+u8 gGamecubeControllerStickCenterX, gGamecubeControllerStickCenterY;
+u8 gGamecubeControllerCStickCenterX, gGamecubeControllerCStickCenterY;
```

Comment:
Can these be turned into structs and be declared as an array size 2 similarly to the other controller structs? It doesn't make sense for these fields to be shared when processing as a for loop.

Reply:
Switched to struct. Seems unnecessary to use arrays here as `stick_x` & `stick_y` are already not arrays

Reply:
`stick_x` and `stick_y` are temporary values that get written to the `data` struct's values. Those do change per controller.

By this, I meant an array of structs, not stick values themselves, since in theory you could have multiple GC controllers plugged in. (That's not currently very practical but it's still good future proofing).

Reply:
My bad, got it fixed now.